<launch>

  <node pkg="my_turtlebot3_vision" name="ros2opencv2_raspi" type="ros2opencv2_raspi.py" output="screen">
    <remap from="input_rgb_image" to="/image_raw" />
    <remap from="input_depth_image" to="/camera/depth/image_raw" />
  </node>
  
</launch>
